### ISSUE TITLE
Layout of management page

### ISSUE THREAD

**Issue by bartaelterman** [inbo/vespa-watch#40]
When a user clicks on "Pest control" in the navbar, he should be redirected to a page that is only accessible for logged in users.
That page contains:

- [x] A table with your observations
- [x] A map with your observations
- [x] A button to record an observation with management actions (see #11 )

 @peterdesmet feel free to add stuff or upload a sketch here.

**Comment by peterdesmet** [comment]
> # Zone X
> 
> `Record a new action` (button)
> 
> ## Alerts
> 
> You have 3 new unmanaged observations since your last visit (of 10 unmanaged)
> 
> ## Map
> 
> - includes all observations assigned to a zone
> - zoomed in to max extend of observations
> - click observation to see pop-up with some info + [edit]() link
>
> ## My observations
>
> Date | Location | Management | ...
> --- | --- | --- | ---
> Recent |  | Reported | [edit]()
> ... |  | Duplicate | [edit]()
> Old |  | Eradicated | [edit]()

* At a larger stage the page could provide some options to filter the observations (in both map and table)
* Unmanaged observations have red background

**Comment by bartaelterman** [comment]
@peterdesmet I've added a basic map and table to this page. Still on the `abstract-observation` branch tough (see #44 )

**Comment by bartaelterman** [comment]
pushed some updates:

- The map and table now only display observations/nests from the zone of the user (based on the foreign key)
- The map automatically pans/zooms to the observations that are displayed
- Nests that have no management action are indicated in red in the table
- Nests in the table are ordered by decreasing date time.

I suggest to move that alerting part to a later release.

@peterdesmet could you review this and add some css magic? I'll assign the ticket to you but feel free to assign it back to me if you have remarks.

**Comment by peterdesmet** [comment]
Blocked by #91

**Comment by peterdesmet** [comment]
Superseded by #321
